lib/run/run.sh

```diff
@@ -75,6 +75,14 @@ bpkg_runner () {
   local cmd="$1"
   shift
 
+  if [ "${cmd:0:1}" = "\"" ]; then
```

Comment:
Could this code be replaced with something like the quote replacement that was added in a different part of this pr?

https://github.com/bpkg/bpkg/pull/170/files#diff-4ffd36319be99fb7f82206db54bcc36ed2fbc2eb5078a685ab4357eae032fd8dR192

```sh
cmd=${cmd#\"}
cmd=${cmd%\"}
```